explorer/client/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -366,18 +368,27 @@ function TransactionView({ txdata }: { txdata: DataType }) {
                                 <ItemView data={typearguments} />
                             </section>
                         )}
-                        {sender && (
+                        {(sender || txKindName === 'TransferSui') && (
                             <section
                                 className={cl([
                                     styles.txcomponent,
                                     styles.txsender,
                                 ])}
                             >
-                                <div className={styles.txaddress}>
-                                    <SendReceiveView data={sendreceive} />
-                                </div>
+                                {txKindName === 'TransferSui' && (
+                                    <div>
+                                        <div>Amount</div>
+                                        <div>{amount ? amount : 0}</div>
+                                    </div>
+                                )}
```

Comment:
```suggestion
                                {amount != null && (
                                    <div>
                                        <div>Amount</div>
                                        <div>{amount}</div>
                                    </div>
                                )}
```

If amount is null, we should hide the component instead of showing it as 0, which may be misleading. You already check `"TransferSui" in data` in  `getTransferSuiAmount`, so no need to duplicate the check here

Reply:
Implemented - thanks